.github/workflows/pypi.yaml

```diff
@@ -0,0 +1,38 @@
+name: Publish pydoctor to PyPI
+on:
+  push:
+    tags: 
+      - '*'
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@master
+
+      - name: Set up Python 3.6
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.6
+      
+      - name: Log system information
+        run: |
+          test -r /etc/os-release && sh -c '. /etc/os-release && echo "OS: $PRETTY_NAME"'
+          python --version
+          python -c "print('\nENVIRONMENT VARIABLES\n=====================\n')"
+          python -c "import os; [print(f'{k}={v}') for k, v in os.environ.items()]"
+
+      - name: Install deps
+        run: |
+          python -m pip install --upgrade pip setuptools wheel
+          python -m pip install -r requirements.txt
+
+      - name: Build pydoctor
+        run: python3 setup.py --quiet build check sdist bdist_wheel
+
```

Comment:
After build, I like to have a Run `ls -al dist/` as a simple way to debug what was created.